### PR TITLE
Sites 1289

### PR DESCRIPTION
**Pull request by jbickar**
# READY FOR REVIEW

# Summary
- Add Feature for [Views Selective Filters](https://www.drupal.org/project/views_selective_filters)

# Needed By (Date)
- Next week should be fine. I just want to make sure we have test coverage before adding the module to the stack.

# Criticality
- How critical is this PR on a 1-10 scale? 3/10
- It impacts ~8 sites to move from Anchorage to ACSF

# Steps to Test

1. Check out this branch
2. Create a new site on ACSF `dev` `cardinald7` with the `stanford` install profile
3. Run this feature and ensure that it passes
4. Run it again to ensure that the cleanup is sufficient

# Affected Projects or Products
- Anchorage migrations to ACSF

# Associated Issues and/or People
- JIRA ticket: `SITES-1289`
- Other PRs: not yet but there will be one to add the module
- Anyone who should be notified? ( @sherakama @minorwm )

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by jbickar** [comment]
Turns out this module already is on the stack and is in all of our profiles. I did not discover that because the name of the project is `views_selective_filters` but the name of the module is `views_filters_selective`.

File: sites/uat/features/low-value/views_filters_selective.feature [hunk 1]
```diff
@@ -0,0 +1,94 @@
+@contrib
```
Comment (sherakama) [inline sites/uat/features/low-value/views_filters_selective.feature]: This is new to me. Where does this come from?

**Review by pookmish — APPROVED** [review]
I remember looking at this awhile back but forgot to approve.

**Review by sherakama — CHANGES_REQUESTED** [review]
Not sure what the `@contrib` statement is. Please revise.

File: sites/uat/features/low-value/views_filters_selective.feature (hunk 1 above)
Comment (jbickar) [reply sites/uat/features/low-value/views_filters_selective.feature]: Looks like copypasta. It's in a large number of Feaures in this `sites/uat` directory. I would have to go back in time about 5 years to remember where it came from.

**Comment by jbickar** [comment]
GTG now?

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**